logos/src/logos/responses.py

```
@@ -211,236 +171,141 @@ def proxy_behaviour(headers, providers, path):
-                continue
-            if part.get("type") != "text":
-                continue
-            txt = part.get("text")
-            if isinstance(txt, str):
-                parts.append(txt)
-        return "\n".join(parts)
-    return ""
+    async def streamer():
+        nonlocal full_text, response, first_response, ttft
+        try:
+            async with httpx.AsyncClient(timeout=None) as client:
```

Ruff (0.14.8)

Probable use of `httpx` call with timeout set to `None`

(S113)